x-pack/plugins/ingest_manager/public/applications/ingest_manager/layouts/with_header.tsx

```diff
@@ -15,18 +15,27 @@ const Page = styled(EuiPage)`
 interface Props extends HeaderProps {
   restrictWidth?: number;
   restrictHeaderWidth?: number;
+  'data-test-subj'?: string;
   children?: React.ReactNode;
 }
 
 export const WithHeaderLayout: React.FC<Props> = ({
   restrictWidth,
   restrictHeaderWidth,
   children,
+  'data-test-subj': dataTestSubj,
```

Comment:
The test subject here is used as the prefix to the header and page test subjects.